src/serac/physics/utilities/functional/tuple_arithmetic.hpp

```diff
@@ -199,23 +159,24 @@ auto chain_rule_tuple_matvec(std::tuple<T...> df_dx, std::tuple<S...> dx)
  * @param[in] A a tuple of values
  * @param[in] B a tuple of values
  * Note: A and B must have the same size, and the type of the ith entry of the returned tuple
- * is given by decltype(std::get<i>(A) + std::get<i>(B))
+ * is given by decltype(serac::get<i>(A) + serac::get<i>(B))
  */
-template <typename... S, typename... T>
-constexpr auto operator+(const std::tuple<S...>& A, const std::tuple<T...>& B)
-{
-  static_assert(sizeof...(S) == sizeof...(T), "Error in operator+: tuple sizes must match");
-  return detail::plus_helper(A, B, std::make_integer_sequence<int, int(sizeof...(S))>{});
-}
+// template <typename... S, typename... T>
```

Comment:
There is a lot of dead code in this file.  reason?

Reply:
With the refactor to use the gpu-enabled `serac::tuple`, much of the arithmetic was put into the `serac::tuple` header directly. We could likely just remove this file completely (might have to move the remaining relevant parts into the `serac::tuple` header).